frontend/src/app/components/ExperimentPage.js

```diff
@@ -18,7 +18,25 @@ import ExperimentPreFeedbackDialog from '../components/ExperimentPreFeedbackDial
 
 const CHANGE_HEADER_ON = 105;
 
+
 export default class ExperimentPage extends React.Component {
+  render() {
+    const { getExperimentBySlug, params } = this.props;
+    const experiment = getExperimentBySlug(params.slug);
```

Comment:
This bit just got moved to the router so we can keep router-knowledge (e.g. like params.slug) out of the individual display components.